providers/core/core.rtcdatachannel.js

```diff
@@ -124,6 +125,11 @@ RTCDataChannelAdapter.prototype.onclose = function (event) {
 RTCDataChannelAdapter.prototype.onmessage = function (event) {
   if (typeof event.data === 'string') {
     this.dispatchEvent('onmessage', {text: event.data});
+  } else if (this.channel.binaryType === 'arraybuffer' &&
```

Comment:
I'm not aware of a relevant bug for this. Not sure if this constitutes a bug on firefox's part or intended behavior.